### ISSUE TITLE
Help to reverse unspent bitcoin sent to bitcoin cash wallet. 

### ISSUE THREAD

**Issue by ashaw1387** [bitcoinjs/bitcoinjs-lib#953]
I accidentally sent bitcoin to bitcoin cash. .05665 plus fees. I've spent days trying to figure out how to verify and sign but I can't figure it out. I'm willing to split with anyone that can help return these funds by broadcasting a raw transaction with myhashID, address and original transaction. The coins are unspent and in a wallet I have no access to. Anyone that can give me help in returning these I will split the BTC with. I used coinbase and they do not share private keys. I have all the info ready when needed. I think I even have the correct format so it shouldn't be much extra work. Please help. Bittrex told me they could help, but won't on anything less then 5k. Shameful really. But my fault. 

If you could email me as well as post here so anyone knows if I've been helped. I apologize if this is not in the correct area. This is my first post.  Thank you!!!! Coinbase to bittrex.

**Comment by dsqr** [comment]
Well, outright scam if they help with the private key if only the amount is high enough. They should have the private key since it's identical.

Not much you can do. Well, wait until your coins are worth 5k or file a lawsuit?

Maybe initiate a little sh*tstorm on social media. This usually helps with old economy companies. ;)

**Comment by dabura667** [comment]
What does this have to do with bitcoinjs-lib?

I am not trying to downplay your situation at all, I just don’t know what you want us to do...?

If you could be more specific, and maybe explain exactly step by step what you would like help on then we can help you.

**Comment by ashaw1387** [comment]
I'm sorry if I posted in the wrong area. I figured I would make a post within the community of people that know this industry far better then I do. More importantly, I also figured people on the forum would understand the back end and know if there was any other way to handle the situation other then using a private key that is being withheld. I agree with you David. I've been sick over this and without pouring my soul into the table, rather cheated. 
I've continued to research the issue and it appears I would need the private key. I've submitted 7 or 8 support tickets with bittrex over the past 30 days. The only response I got at first was asking for my hashid. Provided that and I was then sent a link explaining the $5k threshold on cross chain transactions. I'm assuming the btc key would work with the bch transaction as they share the same account and same address format. 

One thing I have taken away from this is that I understand how blockchain works a lot better then before. 

Anyway, I appreciate both of your replies. And I do apologize if I was out of line posting someplace I shouldn't have. 

All my best,
Andrew Shaw

Sent from my iPhone

> On Dec 13, 2017, at 6:40 PM, Dabura667 <notifications@github.com> wrote:
> 
> What does this have to do with bitcoinjs-lib?
> 
> I am not trying to downplay your situation at all, I just don’t know what you want us to do...?
> 
> If you could be more specific, and maybe explain exactly step by step what you would like help on then we can help you.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
